kumascript/macros/EmbedInteractiveExample.ejs

```diff
@@ -3,28 +3,23 @@
 //
 // Parameters:
 //  $0 - The URL of interactive-examples.mdn.mozilla.net page (relative)
-//  $1 - Optional custom height class to set on iframe element
 //
-//  Example call {{EmbedInteractiveExample("pages/css/animation.html", "taller")}}
+//  Example call {{EmbedInteractiveExample("pages/css/animation.html")}}
 //
 
 const url = kuma.url.resolve(env.interactive_examples.base_url, $0);
-let heightClass = 'is-default-height';
+// Iframe height is acquired from height-data.json file exported by interactive-examples
+const heightData = await mdn.fetchInteractiveExampleHeightData();
 
-if ($0.includes('/js/')) {
-  heightClass = 'is-js-height';
-}
-
-if ($1) {
-  let supportedHeights = ['shorter', 'taller', 'tabbed-shorter', 'tabbed-standard', 'tabbed-taller'];
-  let heightIsSupported = (supportedHeights.indexOf($1) > -1);
-
-  if (heightIsSupported) {
-    heightClass = 'is-' + $1 + '-height';
-  } else {
-    throw new Error(`An unrecognized second size parameter to EmbedInteractiveExample ('${$1}')`);
-  }
+let height;
+if (heightData[$0]) {
+  height = heightData[$0];
+} else if ($0.includes('/js/')) {
+  height = 'js';
+} else {
+  height = 'default';
 }
+const heightClass = 'is-' + height + '-height';
```

Comment:
Let's extract this whole section as `mdn.getInteractiveExampleHeightClass(path)`, so that this macro knows nothing about where the height comes from.

Also, let's keep the $1 parameter for now and pass it to this method, to use as a fallback as long as the `height-data.json` is not available.

Once the `height-data.json` is available, we can deprecate the 2nd parameter.

Reply:
I have moved the whole section to `getInteractiveExampleHeight`. Thanks

Since there was no reply for a long time, I have moved the exact height information to height-data.json. Old classes are no longer compatible, so I didn't apply the second suggestion.